File: expectanim/src/main/java/com/github/florent37/expectanim/core/alpha/AlphaAnimExpectationValue.java
package com.github.florent37.expectanim.core.alpha;

import android.view.View;

/**
 * Created by florentchampigny on 17/02/2017.
 */
public class AlphaAnimExpectationValue extends AlphaAnimExpectation {

    private final float alpha;

    public AlphaAnimExpectationValue(float alpha) {
        this.alpha = alpha;
    }

    @Override
    public Float getCalculatedAlpha(View viewToMove) {
        return alpha;
    }
}

File: expectanim/src/main/java/com/github/florent37/expectanim/core/alpha/AlphaAnimExpectationValue.kt
package com.github.florent37.expectanim.core.alpha

import android.view.View

class AlphaAnimExpectationValue(private val alpha: Float) : AlphaAnimExpectation() {
    override fun getCalculatedAlpha(viewToMove: View): Float? {
        return alpha
    }
}
